lib/player_ai_minimax.rb

```diff
@@ -0,0 +1,60 @@
+class PlayerAiMinimax
+  attr_reader :mark
+
+  def initialize(mark, rules)
+    @mark = mark
+    @rules = rules
+    @corners = [0,2,6,8]
+  end
+
+  def play_move(game_state)
+    board = game_state.game_board
+    return 0 if board.empty?
+    return 4 if opponent_played_corner_for_first_move(board)
+    return corner_move(board) if board.available_moves.size == 8
```

Comment:
Lines 12-14 are all about optimizing the AI player's first move. Maybe move to one private method like `handle_first_move` to make this top-level / public method a little clearer to follow? It seems kinda noisy currently.